spec/requests/citizens/accounts_spec.rb

```diff
@@ -12,5 +25,30 @@
     it 'returns http success' do
       expect(response).to have_http_status(:ok)
     end
+
+    it 'display account holder name' do
+      expect(response.body).to include(applicant_bank_account_holder.full_name)
```

Comment:
Just me wary that (even this is not the case here), if the account holder name contains special characters and they're therefore encoded in the page, the current expectation is likely to fail, so I would suggest to use the [existent helper](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/2e1471215a2e3abc3900dfe33a00484aff35fe79/spec/support/request_helpers.rb#L22-L25) (this applies to the other expectations as well) to perform the assertion against. e.g:

```ruby
 expect(unescaped_response_body).to include(applicant_bank_account_holder.full_name)
```

📝 It's unfortunate that Rails/RSpec used to have a `have_content` helper method on these sort of tests that did exactly that, and they seem to have removed support for it inline with the very confusing (at least to me) Controller Tests vs Requests Tests vs Feature Tests :S